lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -23,7 +23,7 @@ function multiplyFractions(node) {
     return Node.Status.noChange(node);
   }
   const atLeastOneFraction = node.args.some(
-    arg => Node.Type.isOperator(arg, '/'));
+    arg => Node.Type.isFraction(arg));
```

Comment:
ACTUALLY this only works if the function you're passing to `some` only takes one argument. `some` actually passes in 3 arguments to the function you give it: the element, the index, and the array. If your function expects just one argument, the element, that's fine. But if your function can take other optional arguments (like isFraction can take allowParens) then it gets confused because it thinks the index = allowParens. Ugh this is hard to explain but basically it turns out you have to be careful if you use that shortcut!